akka-remote/src/main/resources/reference.conf

```diff
@@ -93,6 +93,28 @@ akka {
       "akka.actor.ActorKilledException" = akka-misc
       "akka.actor.InvalidActorNameException" = akka-misc
       "akka.actor.InvalidMessageException" = akka-misc
+
+      "akka.actor.LocalScope$" = akka-misc
+      "akka.remote.RemoteScope" = akka-misc
+
+      "com.typesafe.config.impl.SimpleConfig" = akka-misc
+      "com.typesafe.config.Config" = akka-misc
+
+      "akka.routing.FromConfig" = akka-misc
+      "akka.routing.DefaultResizer" = akka-misc
+      "akka.routing.BalancingPool" = akka-misc
+      "akka.routing.BroadcastGroup" = akka-misc
```

Comment:
Um, `DaemonMsgCreate.deploy.routerConfig` is a `RouterConfig` and all those subtypes (both pools and groups) can therefore be sent, couldn't find any logic that makes it only possible to send pools, but maybe that just never happens?